src/ripple/app/paths/Pathfinder.h

```diff
@@ -176,6 +176,7 @@ class Pathfinder
 
     Account mSrcAccount;
     Account mDstAccount;
+    Account mEffectiveDst; // The account the paths need to end at
```

Comment:
As I said, I think you should call this `mEffectiveDstAccount` so it's just like `mSrcAccount` and `mDstAccount` (and so you know it's an account when you see it). 

If you just add a little text at the start of this block of variables explaining what the difference is between the "destination account" and the "*effective* destination account" then I'll approve this change for you!